microsoft/testsuites/security/fips.py

```diff
@@ -26,20 +30,64 @@ class Fips(TestSuite):
         priority=3,
     )
     def verify_fips_enable(self, log: Logger, node: Node) -> None:
-        result = node.execute("command -v fips-mode-setup", shell=True)
-        if result.exit_code != 0:
-            raise SkippedException(
-                "Command not found: fips-mode-setup. "
-                f"Please ensure {node.os.name} supports fips mode."
-            )
+        if isinstance(node.os, CBLMariner):
+            # check If It is a non-FIPS image
+            if node.execute("rpm -qa | grep dracut-fips").exit_code != 0:
```

Comment:
I think this command needs sudo

Reply:
No need of sudo here as we are just querying for the package existence.